test/browser/element.vdiff.js

```diff
@@ -179,6 +181,7 @@ describe('element-matches', () => {
 
 });
 
+const expectFailures = true; // Easy switch to create traditional report for debugging
```

Comment:
Added this as an easy way to debug locally - flipping this value makes the tests run "normally" and all fail, resulting in a report with the diffs.